### ISSUE TITLE
min_rig_speed

### ISSUE THREAD

**Issue by cfytrok** [develsoftware/GMinerRelease#567]
How does --min_rig_speed option work?

**Comment by L1TTL3big** [comment]
Yeah, a bit more information on this option would be great. I guess it is related to value produced by rig GPUs not pool hashrate. Anyway if not it would be really nice to have option to restart miner if pool hashrate drops over certain period of time.
